src/lib/errorsHandler.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
```

Comment:
I've read you should use jquery for this type of problem, probably somewhere on stack